src/requirements/data/majors/bio.ts

```diff
@@ -0,0 +1,152 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+
+const bioRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Introductory Biology Cluster',
+    description: 'Take 2 of the following 3: BIOMG 1350, BIOG 1440 OR BIOG 1445, BIOEE 1610 OR BIOSM 1610',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['BIOMG 1350'],
+      ['BIOG 1440', 'BIOG 1445'],
+      ['BIOEE 1610', 'BIOSM 1610']
+    ),
+    operator: 'or',
+    fulfilledBy: 'courses',
+    minCount: 2
+  },
+  {
+    name: 'Investigative Laboratory',
+    description: 'BIOG 1500 OR BIOSM 1500',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['BIOG 1500', 'BIOSM 1500']
+    ),
+    operator: 'and',
+    fulfilledBy: 'courses',
+    minCount: 1
+  },
+  {
+    name: 'Evolutionary Biology and Diversity',
+    description: 'BIOEE 1780 OR BIOEE 1781 OR BIOSM 1780',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['BIOEE 1780', 'BIOEE 1781', 'BIOSM 1780']
+    ),
+    operator: 'and',
+    fulfilledBy: 'courses',
+    minCount: 1
+  },
+  // TODO: temp fix for reqs w options and diff minCount
+  {
+    name: 'General Chemistry Part 1',
+    description: 'CHEM 2070 OR CHEM 2150',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['CHEM 2070', 'CHEM 2150'],
+    ),
+    operator: 'and',
+    fulfilledBy: 'courses',
+    minCount: 1
+  },
+  {
+    name: 'General Chemistry Part 2',
+    description: 'CHEM 2080 OR CHEM 2150',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['CHEM 2080', 'CHEM 2150'],
+    ),
+    operator: 'and',
+    fulfilledBy: 'courses',
+    minCount: 1
+  },
+  {
+    name: 'College Mathematics',
+    description: 'Two courses. One semester of calculus (MATH 1106 OR MATH 1110) '
+      + 'Another course selected from: (MATH 1120 OR MATH 1910) '
+      + 'OR MATH 1105 OR one course in statistics (STSCI 2150 OR BTRY 3010 are preferred)',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    checker: includesWithSubRequirements(
+      ['MATH 1106', 'MATH 1110'],
+      ['MATH 1120', 'MATH 1910', 'MATH 1105', 'STSCI 2150', 'BTRY 3010',
+        'MATH 1710', 'AEM 2100', 'PSYCH 2500', 'ECON 3130', 'SOC 3010'],
+    ),
+    operator: 'and',
+    fulfilledBy: 'courses',
+    minCount: 2
+  },
+  // TODO: problem for reqs w options and diff minCount
```

Comment:
Same for this - do we really want to include this major with so many problems?

Reply:
Not technically problems, just a work around for an edge case that we will build soon